src/Vts/MonteCarlo/AbsorptionWeightingMethods.cs

```diff
@@ -136,10 +130,8 @@ private static double VolumeAbsorptionWeightingDiscrete(PhotonDataPoint previous
         private static double VolumeAbsorbAnalog(PhotonStateType photonStateType)
         {
             var weight = 0.0;
-            if (photonStateType.HasFlag(PhotonStateType.Absorbed))
-            {
-                weight = 1.0;
-            }
+            if (!photonStateType.HasFlag(PhotonStateType.Absorbed)) return weight;

```

Comment:
The variable for weight seems redundant here, we are either returning 1.0 or 0.0, could we make this a return statement?

Reply:
Removed weight and modified into ? : statement, and added header for description.